loama/src/components/SidePanel.vue

```diff
@@ -0,0 +1,64 @@
+<template>
+  <div class="extra-panel" v-if="isOpen">
+    <div>
+      <h2>No Pod yet?</h2>
+      <p>No worries, you can create a pod with a pod provider or even <u>host one</u> yourself! We'll give you some
+        recommendations of hosts you can use:</p>
+      <LoButton v-for="pod in podListData" :key="pod.name" @click="openPodUrl(pod)" :aria-label="'Open ' + pod.name">
+        {{ pod.name }}
+      </LoButton>
+    </div>
+    <LoButton :left-icon="PhX" class="secondary" @click="emit('toggleProvider')">Close Panel</LoButton>
```

Comment:
So, for triggering the secondary variant of the button, it's `:varient="secondary"`
You can change this or keep this, just FYI

```suggestion
    <LoButton :left-icon="PhX" :variant="secondary" @click="emit('toggleProvider')">Close Panel</LoButton>
```